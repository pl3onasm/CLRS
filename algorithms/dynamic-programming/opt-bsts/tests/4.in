.01 .2 .04 .02 .01 .2 .002 .001 .1 .001 .03 .002 .02 .004 .06 .3