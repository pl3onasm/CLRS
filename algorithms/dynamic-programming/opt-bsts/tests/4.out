OBST cost: 2.35

OBST structure:

--| K0 (2)
-| K1 (1)
--| K2 (2)
---| K3 (3)
----| K4 (4)
K5 (0)
---| K6 (3)
----| K7 (4)
--| K8 (2)
-----| K9 (5)
----| K10 (4)
------| K11 (6)
-----| K12 (5)
------| K13 (6)
---| K14 (3)
-| K15 (1)
